x-pack/plugins/security/server/session_management/session_index.ts

```diff
@@ -374,7 +374,7 @@ export class SessionIndex {
     try {
       const response = await this.options.elasticsearchClient.deleteByQuery({
         index: this.aliasName,
-        refresh: true,
+        refresh: true, // TODO: review stateless elasticsearch impact
```

Comment:
__note:__ This particular `refresh: true` occurrence shouldn't be a big deal for overall performance/UX (regular, but low-priority background stale-sessions cleanup job), but we have a few other, more important, `refresh: wait_for` occurrences in this file that we track in https://github.com/elastic/kibana/issues/149338 and plan to investigate if it's feasible to migrate to `refresh: false`.